src/amp-story-player/amp-story-player-impl.js

```diff
@@ -724,16 +723,18 @@ export class AmpStoryPlayer {
     this.prerenderStories_();
     this.initializeVisibleIO_();
 
+    this.prerenderFirstStoryDeferred_.promise.then(() => {
```

Comment:
Is this the right heuristic? The fact that a story is prerendered and the fact that you can use show/add/etc is totally unrelated.
We should make the APIs available ASAP